nextcloud/Caddyfile

```diff
@@ -28,6 +28,11 @@ my-nextcloud-site.com {
 		to /remote.php/{1}/{2}
 	}
 
+	rewrite {
+		r ^/public.php/(.+?)(\/?)$
+		to /public.php/(.+?)(\/?)$
```

Comment:
Take a look to the rest of Caddyfile - it doing the VERY same thing. I've only fixed it to get public sharing by link working. Nothing else.

About why it works - well, IDK. It gives back 404 for me without this rewrite.